binder/src/main/java/io/grpc/binder/internal/ServiceBinding.java

```diff
@@ -193,18 +193,27 @@ private static Status bindInternal(
           bindResult = context.bindService(bindIntent, conn, flags);
           break;
         case BIND_SERVICE_AS_USER:
-          bindResult = context.bindServiceAsUser(bindIntent, conn, flags, targetUserHandle);
+          if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            bindResult = context.bindServiceAsUser(bindIntent, conn, flags, targetUserHandle);
+          } else {
+            return Status.INTERNAL.
+                    withDescription("Cross user Channel requires Android R+");
+          }
           break;
         case DEVICE_POLICY_BIND_SEVICE_ADMIN:
           DevicePolicyManager devicePolicyManager =
-              (DevicePolicyManager) context.getSystemService(Context.DEVICE_POLICY_SERVICE);
-          bindResult =
-              devicePolicyManager.bindDeviceAdminServiceAsUser(
-                  channelCredentials.getDevicePolicyAdminComponentName(),
-                  bindIntent,
-                  conn,
-                  flags,
-                  targetUserHandle);
+                  (DevicePolicyManager) context.getSystemService(Context.DEVICE_POLICY_SERVICE);
+          if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            bindResult = devicePolicyManager.bindDeviceAdminServiceAsUser(
+                            channelCredentials.getDevicePolicyAdminComponentName(),
+                            bindIntent,
+                            conn,
+                            flags,
+                            targetUserHandle);
+          } else {
+            return Status.INTERNAL.
```

Comment:
Same formatting issue here.

I like to automatically reformat *just* the lines my PR changes with a command like: `git diff -U0 HEAD^ | ~/google-java-format-diff.py -v -p1 -i` (https://github.com/google/google-java-format/blob/master/README.md)